src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/utils/UserMdcFilter.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.utils
+import org.slf4j.MDC
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.config.AuthenticationFacade
+import java.io.IOException
+import javax.servlet.Filter
+import javax.servlet.FilterChain
+import javax.servlet.FilterConfig
+import javax.servlet.ServletException
+import javax.servlet.ServletRequest
+import javax.servlet.ServletResponse
+
+@Component
+@Order(1)
+class UserMdcFilter @Autowired constructor(private val authenticationFacade: AuthenticationFacade) : Filter {
+  override fun init(filterConfig: FilterConfig) {
+    // Initialise - no functionality
+  }
+
+  @Throws(IOException::class, ServletException::class)
+  override fun doFilter(request: ServletRequest, response: ServletResponse, chain: FilterChain) {
```

Comment:
Silly question, do we actually use this? e.g with we didn't have that filter what would happen?